docker/models/secrets.py

```diff
@@ -4,14 +4,14 @@
 
 class Secret(Model):
     """A secret."""
-    id_attribute = 'ID'
+    id_attribute = 'Id'
```

Comment:
I'm not sure to understand this change. Can you explain why?

Reply:
I don't remember why I did it, but it's fixed now.
Thanks.